ui/apps/platform/src/Containers/MainPage/Body.tsx

```diff
@@ -183,6 +188,14 @@ function Body({ hasReadAccess, isFeatureFlagEnabled }: BodyProps): ReactElement
                         <Route path={clustersListPath} component={AsyncPFClustersPage} />
                     )}
                     <Route path={systemHealthPath} component={AsyncSystemHealthPage} />
+                    {/* 
+                    TODO - Add any necessary permissions to the following route. The user will need read access to
```

Comment:
This needs discussion with the product team and a Jira ticket has been created so that we don't lose track of this work.